dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx

```diff
@@ -44,19 +45,34 @@ export const DataConfigPanelFields = ({
   handleServiceEdit,
 }: DataConfigPanelFieldProps) => {
   const isAggregation = sectionName === AGGREGATIONS;
+  const { time_field, unit, interval } = dimensionSpan;
 
   // The function hides the click to add button for visualizations included in the const HIDE_ADD_BUTTON_VIZ_TYPES
   const hideClickToAddButton = (name: string) => {
     // returns false when HIDE_ADD_BUTTON_VIZ_TYPES visualizations are not matching with visType.
     if (!isArray(list) || !HIDE_ADD_BUTTON_VIZ_TYPES.includes(visType)) return false;
     // condition for heatmap on the basis of section name
     if (visType === VIS_CHART_TYPES.HeatMap)
-      return name === AGGREGATIONS ? list.length >= 1 : list.length >= 2;
+      return name === AGGREGATIONS
+        ? list.length >= 1
+        : isEmpty(time_field)
+        ? list.length >= 2
+        : list.length >= 1;
     // condition for line and scatter for dimensions section.
     return name === GROUPBY && (list.length >= 1 || !isEmpty(time_field));
   };
 
-  const { time_field, unit, interval } = dimensionSpan;
+  const toolTipTextGenerator = (isAgg: boolean) => {
```

Comment:
In this way we are missing breakdowns here, and this implementations are limited to only be able to serve text for series/dimensions